### DISCUSSION TITLE
Pricing on Activities

### DISCUSSION THREAD

**Discussion by derdoc800** [kimai/kimai#5114] [General]
Is it correct that pricing can only be included with two decimals e.g. 16.16 and not 16.158879 we need this as we need to deduct a certain percentage of the rate as it cannot be invoiced.

Is there maybe a setting which we are not aware of or do we need to raise a separate feature request?

**Comment by kevinpapst** [comment]
> Is it correct that pricing ...

Yes

> Is there maybe a setting which we are not aware of 

No 

> or do we need to raise a separate feature request?

No. But you can explain what exactly your use-case is, which currency you use ... to see if we can find a possible workaround.

**Reply by derdoc800** [reply]
Currency is EUR - unfortunately we have a fixed rate for the customer which we invoice them but due to a procurement company being now in the middle which has a fee we have to deduct a percentage from the fixed rate to be mathematically correct on our revenue and invoice amounts. Unfortuantely this leads to more decimals in the hourly rate on the activity as there percentage does not care about the decimals. I am open for any suggestions. on workarounds

**Reply by kevinpapst** [reply]
I have plans to add a percentage reduction field for invoices, as I was asked for such a feature a few times.

But for now I would manually calculate and deduct the price. 
Or you negotiate a price and not a percentage. 

If the price is fixed per item, how many items do you calculate that it even makes a difference in the end?
Can you give me a practical example that makes sense, for now it doesn't.
